index.js

```diff
@@ -0,0 +1,72 @@
+'use babel';
+
+import { CompositeDisposable } from 'atom';
+import path from 'path';
+
+import handleData from './lib/message.js';
+import servers from './lib/servers.js';
+
+class LinterFlow {
+  config = {
+    pathToFlow: {
```

Comment:
In all the other linter providers, we have named this config `executablePath`, I would suggest you use that too for consistancy